weblate/api/views.py

```diff
@@ -1112,26 +1112,29 @@ def addons(self, request: Request, **kwargs):
 
     def create(self, request: Request, *args, **kwargs):
         """Create a new project."""
+        billing = None
         if not request.user.has_perm("project.add"):
-            self.permission_denied(request, "Can not create projects")
+            if "weblate.billing" in settings.INSTALLED_APPS:
+                from weblate.billing.models import Billing
+
+                try:
+                    billing = Billing.objects.for_user_within_limits(self.request.user)[
+                        0
+                    ]
+                except IndexError:
+                    self.permission_denied(
+                        request, "No valid billing found or limit exceeded."
+                    )
+            else:
+                self.permission_denied(request, "Can not create projects")
```

Comment:
Can you please add test to cover billing not installed and project creation denied?

Reply:
Done.